lib/src/model.dart

```diff
@@ -4630,11 +4654,22 @@ abstract class Nameable {
     }
     return _namePieces;
   }
+
+  String _namePart;
+
+  /// Utility getter/cache for [_MarkdownCommentReference._getResultsForClass].
+  String get namePart {
+    // TODO(jcollins-g): This should really be the same as 'name', but isn't
+    // because of accessors and operators.
+    if (_namePart == null) {
+      _namePart = fullyQualifiedName.split('.').last;
+    }
```

Comment:
Silly, and kind of an aside but I'm curious nonetheless about shorthand (since I do this kind of lazy init all the time).

How crazy is something like this?

`return _namePart ??= fullyQualifiedName.split('.').last;`

Reply:
I'll use that but I now do it only sparingly for really trivial things (e.g. ??= super.foo).  The reason is threefold; one, ??= is kind of obscure and even after programming in Dart for a couple years I still pause for a second when I see it.

Two, if I need to add debugging code for any reason, or want to add something more complex to the statement, I have to add all the boilerplate then rather than concentrate on what I was doing, and then remember to remove it.

And finally, since there are plenty of cases where ??= isn't appropriate stylistically anyway (if we're doing more than one or two lines of work), I prefer to have this idiom look the same wherever I see it.  This way I can concentrate on important things and let this fade into the background by making use of my human brain's ability to process repetitive sequences subconsciously.   Dartdoc isn't quite consistent but I'd like it to be eventually.